x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -912,7 +925,7 @@ export const SaveChangesBottomBar: FunctionComponent = () => {
 function renderHelpText(isOverridden: boolean) {
   if (isOverridden) {
     return (
-      <EuiText size="xs">
+      <EuiText size="xs" data-test-subj="themeOverrideMessage">
```

Comment:
Rather than adding text below the disabled key pad menu, would it be possible to move this text into a tooltip that appears when hovering the key pad menu? I think that may be more consistent with how we handle disabled form elements elsewhere across Kibana.

Reply:
@MichaelMarcialis Would we usually render a tooltip icon next to the component? Similar to these examples:
<img width="402" alt="Screenshot 2023-06-30 at 11 43 02 AM" src="https://github.com/elastic/kibana/assets/103939324/55caf155-adac-4161-b070-3e914c8a74c2">
<img width="412" alt="Screenshot 2023-06-30 at 11 43 54 AM" src="https://github.com/elastic/kibana/assets/103939324/9ae36355-59fb-4894-82ad-f3a31b61822d">

If so, I think the lock icon would make sense.

Reply:
@MichaelMarcialis What do you think of this?
<img width="1051" alt="Screenshot 2023-06-30 at 12 24 12 PM" src="https://github.com/elastic/kibana/assets/103939324/52a3b006-0172-4810-afb8-479af5c0e827">

Reply:
Apologies, I was out on PTO last week. To answer your question, the icon may not even be necessary in this case. I was thinking the tooltip could appear when hovering the disabled key pad menu. Thoughts?

Reply:
@MichaelMarcialis No worries! We decided to render the lock tooltip icon, but am happy to change it if you have a strong preference.

Reply:
Thanks, @jeramysoucy! Assuming it's not a huge lift for ya'll, I say let's go ahead and get rid of the lock icon and have the tooltip appear on hover of the disabled key pad menu. I hate to ask for something so nit-picky, but I think it worth it for the sake of consistency with how we treat other disabled form elements and their accompanying tooltips across Kibana. Let me know if anything is needed on my end.

Reply:
@MichaelMarcialis Should be a quick update. I'll tag you when it's ready.